line-bot-model/src/main/java/com/linecorp/bot/model/action/PostbackAction.java

```diff
@@ -38,6 +39,25 @@
 @JsonTypeName("postback")
 @JsonInclude(Include.NON_NULL)
 public class PostbackAction implements Action {
+    public enum InputOptionType {
+        CLOSE_RICH_MENU("closeRichMenu"),
```

Comment:
FYI: 
We can use @JsonProperty("closeRichMenu") Instead of declaring name and toString().